content/pt-br/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -18,122 +15,91 @@ princípios do Kubernetes, notavelmente o [ciclo de controle](/docs/concepts/#ku
+O padrão operador tem como objetivo capturar o principal objetivo de um operador humano que está gerenciando um serviço ou conjunto de serviços. Operadores humanos que cuidam de aplicativos e serviços específicos possuem um conhecimento profundo de como o sistema deve se comportar, como implantá-lo e como reagir se houver problemas.
 
-As pessoas que executam cargas de trabalho no Kubernetes habitualmente gostam
-de usar automação para cuidar de tarefas repetitivas. O padrão Operador captura
-a forma como pode escrever código para automatizar uma tarefa para além do que
-o Kubernetes fornece.
+As pessoas que executam cargas de trabalho no Kubernetes muitas vezes gostam de usar automação para cuidar de tarefas repetitivas. O padrão do operador captura como você pode escrever código para automatizar uma tarefa além do que o próprio Kubernetes fornece.
 
 ## Operadores no Kubernetes
 
-O Kubernetes é desenhado para automação. *Out of the box*, você tem bastante
-automação embutida no núcleo do Kubernetes. Pode usar
-o Kubernetes para automatizar instalações e executar cargas de trabalho,
-e pode ainda automatizar a forma como o Kubernetes faz isso.
-
-O conceito de {{< glossary_tooltip text="controlador" term_id="controller" >}} no
-Kubernetes permite a extensão do comportamento sem modificar o código do próprio
-Kubernetes.
-Operadores são clientes da API do Kubernetes que atuam como controladores para
-um dado [*Custom Resource*](/docs/concepts/api-extension/custom-resources/)
-
-## Exemplo de um Operador {#exemplo}
-
-Algumas das coisas que um operador pode ser usado para automatizar incluem:
-
-* instalar uma aplicação a pedido
-* obter e restaurar backups do estado dessa aplicação
-* manipular atualizações do código da aplicação juntamente com alterações
-  como esquemas de base de dados ou definições de configuração extra
-* publicar um *Service* para aplicações que não suportam a APIs do Kubernetes
-  para as descobrir
-* simular uma falha em todo ou parte do cluster de forma a testar a resiliência
-* escolher um lider para uma aplicação distribuída sem um processo
-  de eleição de membro interno
-
-Como deve um Operador parecer em mais detalhe? Aqui está um exemplo em mais
-detalhe:
-
-1. Um recurso personalizado (*custom resource*) chamado SampleDB, que você pode
-  configurar para dentro do *cluster*.
-2. Um *Deployment* que garante que um *Pod* está a executar que contém a
-  parte controlador do operador.
+O Kubernetes é desenhado para automação. *Out of the box*, você tem bastante automação integrada ao núcleo do Kubernetes. Você pode usar o Kubernetes para automatizar a implantação e execução de cargas de trabalho, e pode automatizar como o Kubernetes faz isso.
+
+O conceito de {{< glossary_tooltip text="padrão operador" term_id="controller" >}} do Kubernetes permite a extensão do comportamento sem modificar o código do próprio Kubernetes, vinculando controladores a um ou mais recursos personalizados.
```

Comment:
The term_id is `operator-pattern` instead of `controller`

In "vinculando controladores" add glossary_tooltip in controladores, like the original file.